### PR TITLE
Update UG and DG

### PR DESCRIPTION
**Pull request by jessicawyz**
Edit formatting of UG and add diagrams to DG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -202,12 +196,13 @@ These student details can be found in the `test.json` file that we have included
...
 **Tip:** <br>
-To match the results shown in the illustrations within our User Guide, we recommend replacing the contents of the `data/tuteetally.json`file in your directory where `TuteeTally.jar` is saved with the data from [here](https://github.com/AY2324S2-CS2103T-F10-2/tp/tree/master/DummyData/dummydata.json). <br>
+To match the results shown in the illustrations within our User Guide, we recommend replacing the contents of the `data/tuteetally.json`file in your directory where `TuteeTally.jar` is saved with the data from [here](https://github.com/AY2324S2-CS2103T-F10-2/tp/tree/
+  /DummyData/dummydata.json). <br>
 This will ensure your outputs align with the examples provided in our guide. <br>
 
 </box>
 
-### View student statistics: `view`
+### View student statistics: `view -stats`
```
Comment (MarcusGitty) [inline docs/UserGuide.md:205]: Thanks for catching this

File: docs/UserGuide.md [hunk 2]
```diff
@@ -341,8 +337,18 @@ This will add an exam to a student by searching for their `ID`. Time is an optio
 **Format 1:** `addexam -id {ID} -exam {EXAMNAME} -date {DATE}` <br>
 **Format 2:** `addexam -id {ID} -exam {EXAMNAME} -date {DATE} -time {TIME}`
 
+**Notes on Fields for add exam command**
+
+| Field        | Prefix   | Required | Caveats                                                                                                                                                                     |
+|--------------|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **uniqueId** | `-id`    | Yes      | Maximum 6 digits allowed but you can omit any leading 0s.<br> e.g. `1`, `01`, `000001` are all valid id inputs for the id `#000001`                                         |
```
Comment (MarcusGitty) [inline docs/UserGuide.md:344]: I'm concern about the phrasing of  "Maximum 6 digits allowed", cuz iD::isValid method also checks if the value is between 1 and 999999 (both inclusive). so 7 or more digits can be valid due to leading 0's

I scared for the PE, they might just test and say, but the below is valid so it's a bug there's a mismatch between UG and app.

ID 000000001 or
ID 00000009999 (if id 9999 exist)

**Review by MarcusGitty — CHANGES_REQUESTED** [review]
LGTM, just have a comment on ID !

File: docs/UserGuide.md (hunk 2 above)
Comment (jessicawyz) [reply docs/UserGuide.md:344]: ok will change to positive integer with maximum 6 digits with leading 0s optional

**Comment by jessicawyz** [comment]
I edited the phrasing for the places that explains about the id, hopefully it is clearer now

**Review by MarcusGitty — APPROVED** [review]
LGTM, thanks for making ID clearer!

**Pull request merged**